cmd/embedded-cluster/install.go

```diff
@@ -179,7 +181,73 @@ func checkLicenseMatches(c *cli.Context) error {
 	}
 
 	return nil
+}
+
+func checkAirgapMatches(c *cli.Context) error {
+	rel, err := release.GetChannelRelease()
+	if err != nil {
+		return fmt.Errorf("failed to get release from binary: %w", err) // this should only be if the release is malformed
+	}
+	if rel == nil {
+		return nil // TODO make this an error when done testing
+	}
+
+	// read file from path
+	rawfile, err := os.Open(c.String("airgap-bundle"))
+	if err != nil {
+		return fmt.Errorf("failed to open airgap file: %w", err)
+	}
+	defer rawfile.Close()
+
+	appSlug, channelID, airgapVersion, err := airgap.AirgapBundleVersions(rawfile)
```

Comment:
```suggestion
	appSlug, channelID, versionLabel, err := airgap.ChannelReleaseMetadata(rawfile)
```

Reply:
Are you suggesting I rename the function?

Reply:
I do like this new name better